src/app/auth/pages/login-page/login-page.component.ts

```diff
@@ -48,15 +30,29 @@ export class LoginPageComponent implements OnInit {
             return;
         }
 
-        const { login, password } = this.form.value;
-        const loggedIn = this.authService.login(login, password);
+        this.isSubmitted = true;
+
+        const { email, password } = this.form.value;
+        const loggedIn = this.authService.login(email, password);
 
         if (loggedIn) {
-            this.form.reset();
-            this.router.navigate(['/youtube']);
-            this.isSubmited = false;
+            this.handleSuccessfulLogin();
         } else {
-            this.isSubmited = false;
+            this.handleFailedLogin();
         }
     }
+
+    handleSuccessfulLogin() {
+        this.form.reset();
+        this.router.navigate(['/youtube']);
```

Comment:
navigate should be the last one line (after this.isSubmitted = false;)

Reply:
Fixed.
```
handleSuccessfulLogin() {
        this.form.reset();
        this.isSubmitted = false;
        this.router.navigate(['/youtube']);
 }
```